modules/ossm-extensions-migration-overview.adoc

```diff
@@ -0,0 +1,90 @@
+////
+This module included in the following assemblies:
+*service_mesh_/v2x/ossm-extensions.adoc
+////
+:_content-type: CONCEPT
+[id="ossm-extensions-migration-overview_{context}"]
+= Migrating from `ServiceMeshExtension` to `WasmPlugin` resources
+
+The `ServiceMeshExtension` API is deprecated as of {SMProductName} version 2.2 and will be removed in a future release. If you are using the `ServiceMeshExtention` API, you must migrate to the `WasmPlugin` API for your extensions.
+
+The APIs are very similar. The migration consists of two steps:
+
+. Renaming your plugin file and updating the module packaging.
+
+. Creating a `WasmPlugin` resource that references the updated container image.
+
+[id="ossm-extensions-migration-api-changes_{context}"]
+== API changes
+
+The new `WasmPlugin` API is quite similar to the `ServiceMeshExtension`, with a few differences, especially in the field names:
+
+
+.Field changes between `ServiceMeshExtensions` and `WasmPlugin`
```

Comment:
See my earlier comment about filenames, and consider updating all such instances in this section too.